configs/components/rubygem-yard.rb

```diff
@@ -1,6 +1,6 @@
 component 'rubygem-yard' do |pkg, settings, platform|
-  pkg.version '0.9.26'
-  pkg.md5sum '0eff29921e52a7a422eee1574ede3d1d'
+  pkg.version '0.9.27'
+  pkg.md5sum 'ed897c4c7e09714d376a359e5e52d70b'
```

Comment:
This version of yard requires [webrick ~> 1.27.0](https://rubygems.org/gems/webrick), so we'll need to add it as a new rubygem component and then add it to both the bolt-runtime and _shared-pe-bolt-server projects.

Reply:
https://github.com/lsegal/yard/commit/8dd8d32f370c4809d3f08c653159524b778f00ac Yeah they just added that

Reply:
Added component and added to both projects